packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -254,26 +311,61 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
           key: `${option.label}-${index}`,
           isSelected,
           onClick: () => {
-            !option.disabled && onChange?.(option.value);
-            close();
+            !option.disabled &&
+              onChange?.(
```

Comment:
So now for single selectable Select it will call `onChange` with value and `added` as second parameter? If yes, that's can be a breaking change.

Reply:
Yes, it will, but I did not update interface that value is coming and also did not update any examples with it (except unit test).
Why that would be a breaking change? It comes as invisible parameter.

Reply:
If someone is passing a function that accepts two parameters, second one were always getting undefined and nothing happened but now it will get "added". I don't remember exactly but I have read somewhere to avoid doing that.

Reply:
what? Why they would accept 2nd parameter from `onChange`, if they know its giving only one?
Or I misunderstood the use case

Reply:
If they are reusing some function e.g. `handleItem(value: string, forceUpdate?: boolean)`, second parameter previously would always be undefined/falsie but not it will be truthy.

Reply:
sounds pretty weird...
But I guess I can try to add 2nd parameter only when multiple is enabled..

Reply:
fixed